model/task/task_test.go

```diff
@@ -3140,6 +3141,148 @@ func TestArchiveMany(t *testing.T) {
 	}
 }
 
+func TestArchiveManyFailedOnly(t *testing.T) {
+	require.NoError(t, db.ClearCollections(Collection, OldCollection))
+	et1 := Task{
+		Id:                    "et1",
+		Status:                evergreen.TaskFailed,
+		Execution:             2,
+		LatestParentExecution: 2,
+		Aborted:               true,
+		Version:               "v",
+	}
+	assert.NoError(t, et1.Insert())
+	et2 := Task{
+		Id:                    "et2",
+		Status:                evergreen.TaskSucceeded,
+		Execution:             2,
+		LatestParentExecution: 2,
```

Comment:
Sorry for the late thought -- can we remove LPE here and ensure that the tests still work? This is the existing backwards incompatible case so I want to make sure this doesn't break restarts.